README.md

```diff
@@ -24,7 +24,7 @@ $ npm install mathball --save
 
 - **Number Checkers** - [isArmstrong][3] - [isAutomorphic][4] - [isCarmichael][43] - [isDeficient][42] - [isFrugal][47] - [isHarshad][45] - [isKaprekar][23] - [isLucky][24] - [isNeon][5] - [isPalindrome][6] - [isPerfect][27] - [M.check('numberCheckers')(Number)][7] - [isSmith][26] - [isStormer][52] -[isHoax][54]
 
-- **Find nth Number** - [bell][32] - [catalan][22] - [fibonacci][8] - [find('nthNumber')(Number)][21] - [magic][9] - [nCr][48] - [magic][9] - [padovan][41] - [smart][25] - [triangular][93] - [ugly][56] 
+- **Find nth Number** - [bell][32] - [catalan][22] - [fibonacci][8] - [find('nthNumber')(Number)][21] - [magic][9] - [nCr][48] - [magic][9] - [padovan][41] - [smart][25] - [triangular][93] - [ugly][56] - [nCrCatalan][118]
```

Comment:
You **don't** need to make changes to this.
What this issue requires is, to make changes to the already existing `catalan/index.js` file.

You need to import `nCr` function there, and use it, reducing redundancy.
`nCr` is same as `binomialCoeff`